packages/editor/src/editor/sidebar/fields/InputFieldWithBrowser.tsx

```diff
@@ -48,12 +30,17 @@ export const InputFieldWithBrowser = ({
             ref={inputRef}
             value={value}
             onChange={e => onChange(e.target.value)}
-            onSelect={handleTextSelection}
+            onSelect={() => handleTextSelection(inputRef, setSavedSelection)}
             onBlur={onBlur}
             placeholder={options?.placeholder}
           />
-          {browsers.some(b => b === 'CMS') && inputRef.current?.selectionStart !== inputRef.current?.selectionEnd && savedSelection && (
-            <AddCmsQuickActionPopover value={getSelectedText()} savedSelection={savedSelection} inputRef={inputRef} onChange={onChange} />
+          {canPopoverBeRendered(inputRef, savedSelection) && savedSelection && browsers.some(b => b === 'CMS') && (
+            <AddCmsQuickActionPopover
+              value={getSelectedText(inputRef, savedSelection)}
+              savedSelection={savedSelection}
```

Comment:
why is this called savedSelection?
What is saved? isn't this just the selection?